### ISSUE TITLE
Subscriptions ready

### ISSUE THREAD

**Issue by yogiben** [inProgress-team/react-native-meteor#40]
Has there been any attempt to listen to whether the sub is ready for this kind of use case:

```
isReady ? <TodosList todos={todos] /> : <Loading />
```

This would be a lovely addition to 1.0 #33

**Comment by Mokto** [comment]
Yes I will add this soon ;)

Thanks.

**Comment by Mokto** [comment]
Sorry I have no time to do this right now. Maybe next week.

PRs are welcome ! ;)

**Comment by Mokto** [comment]
@yogiben Actually I took the time to to what you asked.

It is available in 1.0.0-beta25.

  ```javascript
getMeteorData() {
    const handle = Meteor.subscribe('todos');

    return {
      todosReady: handle.ready()
    };
  }
```
